pyproject.toml

```diff
@@ -6,16 +6,16 @@ authors = ["CTDS UChicago <cdis@uchicago.edu>"]
 license = "Apache-2.0"
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.9"
 fastapi = "^0.65.2"
-uvicorn = {extras = ["standard"], version="^0.11.2"}
+uvicorn = {extras = ["standard"], version=">=0.11.8,<1.0.0"}
 gunicorn = "^20.0.4"
 gino = {version = "^1.0.0", extras = ["starlette"]}
 alembic = "^1.3"
 psycopg2-binary = "^2.8"
-gen3authz = "^1.0.0"
+gen3authz = "^1.5.1"
 indexclient = "^2.1.0"
-httpx = "^0.12.1"
+httpx = ">=0.20.0,<1.0.0"
```

Comment:
httpx `0.20.0` did have a [backwards incompatible change to not automatically follow redirects](https://github.com/uc-cdis/gen3authz/pull/38). I don't think that impacts MDS at all, but just thought I'd note it.